.gitlab/build_agent.yaml

```
@@ -10,10 +11,15 @@
     # disable kitchen and e2e tests
     RUN_KITCHEN_TESTS: "false"
     RUN_E2E_TESTS: "off"
+  # Override AGENT_BRANCH for release branches
+  rules:
+    - if: $CI_MERGE_REQUEST_TARGET_BRANCH_NAME =~ /^7\.\d+\.x$/
```

```suggestion
    - if: $CI_MERGE_REQUEST_TARGET_BRANCH_NAME =~ /^[0-9]+\.[0-9]+\.x$/
```
nit can we match the same as the agent CI? https://github.com/DataDog/datadog-agent/blob/main/.gitlab-ci.yml#L273